apps/meteor/server/methods/saveUserProfile.ts

```diff
@@ -19,6 +20,7 @@ import { compareUserPasswordHistory } from '../lib/compareUserPasswordHistory';
 const MAX_BIO_LENGTH = 260;
 const MAX_NICKNAME_LENGTH = 120;
 
+// eslint-disable-next-line complexity
```

Comment:
I'd suggest not disabling the warning here, it'd be better to have it explicit, so we can refactor it as soon as possible.

Reply:
Ok, I'll remove this line!